src/dense/svds/gesvd/gesvd.jl

```diff
@@ -0,0 +1,111 @@
+include("magmaCgesvd.jl")
```

Comment:
why are we still have so many manual wrappers?

Reply:
why we still have so many manual wrappers so far? can we just use `eval` to generate the wrappers? 
If you still can't understand what is eval, check the implementation here:

https://github.com/JuliaLang/julia/blob/master/stdlib/LinearAlgebra/src/blas.jl

Reply:
Already used eval to generate the wrappers. See src/dense/svds/gesvd/gesvd2.jl please.
I followed the coding style of Julia stdlib.
If no severe problem, I may clear the other redundant files later.